cmd/filegen/Dockerfile

```diff
@@ -0,0 +1,19 @@
+
+FROM golang:1.13.6 as builder
+MAINTAINER ElrondNetwork
+
+WORKDIR /go/elrond-deploy-go
+COPY . .
+WORKDIR /go/elrond-deploy-go/cmd/filegen
+RUN GO111MODULE=on go mod vendor
+RUN go build
+
+# Second stage
+FROM ubuntu:18.04
+COPY --from=builder "/go/elrond-deploy-go/cmd/filegen" "/go/elrond-deploy-go/cmd/filegen"
+WORKDIR /data
+ENTRYPOINT ["/go/elrond-deploy-go/cmd/filegen/filegen"]
+
+#RUN ["./filegen", "-total-supply", "1000000000000000000000000000", "-num-of-shards", "3", "-num-of-nodes-in-each-shard", "12", "-consensus-group-size", "12", "-num-of-observers-in-each-shard", "1", "-num-of-metachain-nodes", "12", "-metachain-consensus-group-size", "12", "-num-of-observers-in-metachain", "1", "-chain-id", "testnet", "-hysteresis", "0.0"]
```

Comment:
Do we still need those comments? As a rule of thumb we do not keep comments as they might induce ambiguity in the overall code and confusion for the next  developers/reviewers. 
If there is a case in which the work is partially done because of a constraint, we add TODO markups describing something like "TODO uncomment this when component such-and-such is fully developed" or things like this.

Reply:
thanks, good to know, will update/remove it